.huskyrc

```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
+    "pre-commit": "npx eslint ."
```

Comment:
We already have the `npm run lint` script(s) that we can leverage here.

Reply:
@swissspidy good catch. Will make the change.